modules/customLoggingApi/apiIntegrations.tf

```diff
@@ -0,0 +1,20 @@
+resource "aws_api_gateway_integration" "sqs-integration" {
+  http_method = aws_api_gateway_method.proxy.http_method
+  resource_id = aws_api_gateway_resource.proxy.id
+  rest_api_id = aws_api_gateway_rest_api.logging_gateway.id
+  type = "AWS"
+  integration_http_method = "POST"
+  uri = "arn:aws:apigateway:eu-west-2:sqs:path/${aws_sqs_queue.custom_log_queue.name}"
+  credentials = aws_iam_role.custom-logging-api-gateway-role.arn
+
+  request_parameters = {
+    "integration.request.header.Content-Type" = "'application/x-www-form-urlencoded'"
+  }
+
+  request_templates = {
+    "application/json" = <<EOF
+Action=SendMessage&MessageBody=$input.json('$')
+EOF
+  }
```

Comment:
Is this format the standard for ```EOF```? Can we change this to have better indentation?